content/en/blog/2022-03-21-tools-to-manage-helm-declaratively.md

```diff
@@ -39,39 +39,46 @@ The Kubernetes ecosystem has produced numerous projects of various styles to hel
-* [Argo CD](https://github.com/argoproj/argo-cd) - The [Argo](https://argoproj.github.io/) project defines itself as providing "Open source tools for Kubernetes to run workflows, manage clusters, and do GitOps right." Argo CD is focused on declarative continuous delivery and has the ability to work with Helm charts.
+* [Flux Helm controller] - [Flux](https://fluxcd.io/) is a collection of projects that enable GitOps. One of the components provides a GitOps method to manage Helm releases.
+* [Argo CD] - The [Argo](https://argoproj.github.io/) project defines itself as providing "Open source tools for Kubernetes to run workflows, manage clusters, and do GitOps right." Argo CD is focused on declarative continuous delivery and has the ability to work with Helm charts.
 
 ### Other Projects
 
 There are many projects beyond the CNCF projects you can use to help you manage your Helm releases. The following set is an example and not exhaustive.
 
-* [Helmfile](https://github.com/roboll/helmfile) - A declarative spec for deploying Helm charts.
-* [Captain](https://github.com/alauda/captain) - A Helm controller
-* [Helm Provider for Terraform](https://github.com/hashicorp/terraform-provider-helm) - Enables you to manage Helm charts through Terraform
-* [Orkestra](https://azure.github.io/orkestra/) - Built on other tools in this list, Orkestra adds a robust dependency graph for a related group of Helm releases and their subcharts, as well as a reverse DAG for specifying dependency requirements for rollbacks.
-* [Fleet](https://github.com/rancher/fleet) - A GitOps tool chain that works with Kubernetes manifests, Helm charts, and Kustomize.
+* [Helmfile] - A declarative spec for deploying Helm charts.
+* [Captain] - A Helm controller.
+* [Terraform Helm provider] - Enables you to manage Helm charts through Terraform.
+* [Orkestra] - Built on other tools in this list, Orkestra adds a robust dependency graph for a related group of Helm releases and their subcharts, as well as a reverse DAG for specifying dependency requirements for rollbacks.
+* [Fleet] - A GitOps tool chain that works with Kubernetes manifests, Helm charts, and Kustomize.
 
 ### High-Level Tool Comparison
 
 There are some differences between the tools we've looked at so far. The following table provides some insight into their differences. This is not exhaustive and you should evaluate any tools you use yourself.
 
-| | Retains Helm release info | Supports Helm hooks | OCI support |
-| -- | -- | -- | -- |
-| Helmfile | ✅ | ❓[^1] | 🚫 |
-| Terraform Helm provider | ✅ | ✅[^2] | 🚫[^3] |
-| Captain Helm controller | ✅ | ❓[^4] | 🚫[^5] |
-| Flux Helm controller | ✅ | ✅  | 🚫[^6] |
-| ArgoCD | 🚫 | 🚫[^7] | ✅[^8] |
-| Orkestra | ✅ | ✅ | 🚫[^9] |
-| Fleet | ✅ | ✅ | 🚫[^10] |
+| | Retains Helm release info | Supports Helm hooks | OCI support | Requires Helm binary |
```

Comment:
Can we make this a negative statement instead?  instead of "Requires Helm binary" maybe "Does not require Helm binary" then those can be ✅ instead of 🚫? This is my bias, thinking it's a positive thing that tools like this can run on scratch and not depend on shelling out. But I think that's your reason for adding this column back, right?

Reply:
Agreed.